README.rst

```diff
@@ -0,0 +1,144 @@
+Spinal Cord Toolbox
```

Comment:
great idea to make it an RST-- it will also become useful if (one day) SCT is hosted on pypi (because they only accept RST)

Reply:
fyi they accept markdown. You just have to set

```
  long_description_content_type='text/markdown',
```

see e.g. https://test.pypi.org/project/spinalcordtoolbox-data-optic-models/ from https://github.com/sct-data/optic_models/blob/master/setup.py